scripts/check_driver.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# https://github.com/aquadzn
+# ```bash check_driver.sh```
+
+
+check_codec() {
+    if $(vainfo |& grep -q -e "H265" -e "HEVC")
```

Comment:
I don't know if it is a sufficient condition to have H265|HEVC inside the vainfo to actually support the H265 codec. These are 2 regex I used to check H265 and H264:

- `/VAProfileH264Main.*:\t(VAEntrypoint\w*)/`
- `/VAProfile(HEVC|H265)Main.*:\t(VAEntrypoint\w*)/`